servicetalk-http-utils/src/main/java/io/servicetalk/http/utils/BeforeFinallyOnHttpResponseOperator.java

```diff
@@ -254,28 +255,4 @@ public void onComplete() {
             // Ignore.
         }
     }
-
-    private static final class RunnableTerminalSignalConsumer implements TerminalSignalConsumer {
```

Comment:
I realize I requested to make the conversion classes internal when these types were recently introduced, but if we are going to have to duplicate the code lets just expose it so we can share.

Reply:
Sad that we can not place it in some `-internal` module due to the dependencies between modules to share it with `http-utils`.

Reply:
agreed the API needs to be public in `concurrent-api` so we would need to push the APIs down to the `concurrent` module in order to put the utility classes in an `-internal` module.